src/domains/wallet-app/api/webview-payment-wallet/v1/get_psps_policy.xml.tpl

```diff
@@ -0,0 +1,121 @@
+            <set-url>@("https://${ecommerce_hostname}/pagopa-ecommerce-payment-methods-service/payment-methods/" + context.Request.MatchedParameters["paymentMethodId"])</set-url>
+            <set-method>GET</set-method>
+            <set-header name="x-client-id" exists-action="override">
+                <value>IO</value>
+            </set-header>
+        </send-request>
+
+        <choose>
+            <when condition="@(((IResponse)context.Variables["paymentMethodsResponse"]).StatusCode == 404)">
+                <return-response>
+                    <set-status code="404" reason="Not Found" />
+                    <set-header name="Content-Type" exists-action="override">
+                        <value>application/json</value>
+                    </set-header>
+                    <set-body>
+                        {
+                            "title": "Payment method not found",
+                            "status": 404,
+                            "detail": "Couldn't find the provided payment method"
+                        }
+                    </set-body>
+                </return-response>
+            </when>
+            <when condition="@(((IResponse)context.Variables["paymentMethodsResponse"]).StatusCode != 200)">
+                <return-response>
+                    <set-status code="502" reason="Bad Gateway" />
+                    <set-header name="Content-Type" exists-action="override">
+                        <value>application/json</value>
+                    </set-header>
+                    <set-body>
+                        {
+                            "title": "Error retrieving eCommerce payment methods",
+                            "status": 502,
+                            "detail": "There was an error retrieving eCommerce payment methods"
+                        }
+                    </set-body>
+                </return-response>
+            </when>
+        </choose>
+        <set-variable name="paymentMethodsResponseBody" value="@(((IResponse)context.Variables["paymentMethodsResponse"]).Body.As<JObject>())" />
+        <set-variable name="paymentMethodTypeCode" value="@((string)((JObject) context.Variables["paymentMethodsResponseBody"])["paymentTypeCode"])" />
+
+        <set-header name="Content-Type" exists-action="override">
+            <value>application/json</value>
+        </set-header>
+
+        <set-body>
+            @{
+                var requestBody = new JObject();
+                requestBody["touchpoint"] = "IO";
+                requestBody["paymentMethod"] = (string) context.Variables["paymentMethodTypeCode"];
+                requestBody["paymentAmount"] = 1;
+                requestBody["isAllCcp"] = false;
+                requestBody["primaryCreditorInstitution"] = "";
+
+                var transfer = new JObject();
+                transfer["creditorInstitution"] = "";
+                transfer["digitalStamp"] = false;
```

Comment:
Are you sure we can set this value to false by default?

Reply:
Yes, this this is the value agreed upon with GEC for onboarding :)